projects/services-ogc/src/lib/owc/types/owc-json.ts

```diff
@@ -68,21 +59,6 @@ export interface IOwsContext extends GeoJSON.FeatureCollection<GeoJSON.GeometryO
   [k: string]: any;
 }
 
-export function isIOwsContext(object: any): object is IOwsContext {
```

Comment:
Removing an exported function is also a breaking Change.

Reply:
Have been restored! This was an unwanted regression - of course we want to keep those type-checkers. Thanks for noticing!